lib/crapto/examples/repeating_key_xor_solve.nit

```diff
@@ -0,0 +1,24 @@
+import base64
+import crapto
+
+# Read the cipher from the file
+var reader = new FileReader.open("repeating_key_xor_cipher.txt")
```

Comment:
When jenkins (or other) will try to open the file it will fail because the working directory will be likely `tests/`.

A solution is to accept the file as an argument and create a file `tests/repeating_key_xor_solve.args` with the content `lib/crapto/examples/repeating_key_xor_cipher.txt`